### PR TITLE
Revert "Revert "Add release notes for DevTools 2.15.0 and 2.16.0""

### PR DESCRIPTION
**Pull request by domesticmouse**
Reverts flutter/website#7424 so @sfshaza2 can make edits and land.

### CODE REVIEW

**Comment by domesticmouse** [comment]
Please edit as required @sfshaza2

**Comment by domesticmouse** [comment]
PTAL @sfshaza2

**Comment by domesticmouse** [comment]
@khanhnwin we need this PR landed before the freeze.

**Comment by sfshaza2** [comment]
I'm happy to land this, but I'm a bit confused as to the process.

**Comment by khanhnwin** [comment]
@domesticmouse Noted! @sfshaza2 once you add your edits to the release notes we can land it?

**Comment by domesticmouse** [comment]
@sfshaza2 happy to jump on a VC to discuss

File: src/development/tools/devtools/release-notes/release-notes-2.15.0-src.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+Dart & Flutter DevTools - A Suite of Performance Tools for Dart and Flutter
+
+## General Updates
+* Made performance improvements to all tables in DevTools (logging view, network profiler, CPU profiler etc.) - [#4175](https://github.com/flutter/devtools/pull/4175)
+
+## Performance Updates
+* Added outlines to each layer displayed in the Raster Metrics tool - [#4192](https://github.com/flutter/devtools/pull/4192)
+![raster-metrics-layer-outlines]({{site.url}}/development/tools/devtools/release-notes/images-2.15.0/image1.png "raster metrics layer outlines")
+* Fix a bug with loading offline data - [#4189](https://github.com/flutter/devtools/pull/4189)
```
Comment (sfshaza2) [inline src/development/tools/devtools/release-notes/release-notes-2.15.0-src.md:11]:
```suggestion
* Added outlines to each layer displayed in the Raster Metrics
   tool&mdash;[#4192](https://github.com/flutter/devtools/pull/4192)
![raster-metrics-layer-outlines]({{site.url}}/development/tools/devtools/release-notes/images-2.15.0/image1.png "raster metrics layer outlines")
* Fix a bug with loading offline data&mdash;[#4189](https://github.com/flutter/devtools/pull/4189)
```

File: src/development/tools/devtools/release-notes/release-notes-2.16.0-src.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# DevTools 2.16.0 release notes
+
+Dart & Flutter DevTools - A Suite of Performance Tools for Dart and Flutter
+
+## General Updates
+This release included several bug fixes and improvements, as well work
+towards some new features that are coming soon! See the full list of commits for DevTools 2.16.0
+[here](https://github.com/flutter/devtools/blob/master/CHANGELOG.md#2160).
```
Comment (sfshaza2) [inline src/development/tools/devtools/release-notes/release-notes-2.16.0-src.md:8]:
```suggestion
towards some new features that are coming soon!
See the full list of commits for DevTools 2.16.0
[in the changelog](https://github.com/flutter/devtools/blob/master/CHANGELOG.md#2160).
```

File: src/development/tools/devtools/release-notes/release-notes-2.16.0-src.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+## Changelog
+More details about changes and fixes are available in our
```
Comment (sfshaza2) [inline src/development/tools/devtools/release-notes/release-notes-2.16.0-src.md:11]:
```suggestion
More details about changes and fixes are available in the DevTools
```

**Review by sfshaza2 — APPROVED** [review]
lgtm, with some minor changes. (For example, never link on "here")

**Comment by domesticmouse** [comment]
FYI @kenzieschmoll =)

**Pull request merged**